js/ad_manager_vast.js

```diff
@@ -305,6 +349,46 @@ OO.Ads.manager(function(_, $) {
     };
 
     /**
+     * Determine if a Vast ad is skippable, and if so, when the skip ad button should be displayed.
+     * Notifies AMC of the result.
+     * @private
+     * @method Vast#calculateSkipAdOffset
+     * @param {object} adWrapper The current Ad's metadata.
+     */
+    var calculateSkipAdOffset = _.bind(function(adWrapper) {
```

Comment:
when it's name starts with calculate, it makes me think it should return a value. Maybe name it initSkipAdOffset?